### ISSUE TITLE
Browser spell checking doesn't work in code editor

### ISSUE THREAD

**Issue by bkaes** [Codewars/codewars.com#67]
I guess this is due to a special code editor, but it's a little bit of a hassle, especially for descriptions, where you want to check the spelling thoroughly.

**Comment by jhoffner** [comment]
Yeah the idea solution to this is to provide a toggle to turn off the code editor completely. This would also make for a much better mobile experience.
